iam/examples/keycloak_kafka/docker-compose.yml

```diff
@@ -0,0 +1,180 @@
+version: "3.8"
+services:
+  keycloak:
+    build:
+      context: ../../keycloak
+    depends_on:
+      - kafka
+      - postgres
+    restart: always
+    environment:
+      JAVA_OPTS_APPEND: "-Dkeycloak.profile.feature.impersonation=disabled -Dkeycloak.profile.feature.upload_scripts=enabled"
+      KEYCLOAK_USER: admin
+      KEYCLOAK_PASSWORD: admin
+      DOJOT_ROOT_URL: "http://localhost:8080"
+      DOJOT_CUSTOM_REALM_REP_FILE: "/opt/dojot/customRealmRepresentation.json"
+      # DOJOT_ADMIN_PASSWORD: aA1!bB2@cC3#
```

Comment:
if `DOJOT_ADMIN_PASSWORD` has not been defined, what is the behavior? I think that ideal is cant get up.

Reply:
Done. This attribute has become mandatory for Keycloak to work!

When trying to create a new Realm, this password will be confronted with the password policies defined in the realm customization json file, if the password is not strong, a message will be displayed on the screen.
In this case, the platform administrator will have to restart the keycloak and set a password that satisfies the password policies.